app/views/RoomView/LeftButtons.tsx

```diff
@@ -13,11 +13,37 @@ const styles = StyleSheet.create({
 	}
 });
 
+interface IRoomLeftButtonsProps {
+	tmid?: string;
+	unreadsCount: number;
+	navigation: StackNavigationProp<ChatsStackParamList>;
+	baseUrl: string;
+	userId: string;
+	token: string;
+	title: string;
+	t: string;
+	theme: string;
+	goRoomActionsView: Function;
+	isMasterDetail: boolean;
+}
+
 const LeftButtons = React.memo(
-	({ tmid, unreadsCount, navigation, baseUrl, userId, token, title, t, theme, goRoomActionsView, isMasterDetail }) => {
+	({
+		tmid,
+		unreadsCount,
+		navigation,
+		baseUrl,
+		userId,
+		token,
+		title,
+		t,
+		theme,
+		goRoomActionsView,
+		isMasterDetail
+	}: IRoomLeftButtonsProps) => {
 		if (!isMasterDetail || tmid) {
-			const onPress = useCallback(() => navigation.goBack());
-			const label = unreadsCount > 99 ? '+99' : unreadsCount || ' ';
+			const onPress = useCallback(() => navigation.goBack(), []);
```

Comment:
I only added the empty array because typescript throw an error if we do not pass any param as second dependence.
In my most recently test, everything still works as usual.


![Screen Shot 2021-12-20 at 15 53 35](https://user-images.githubusercontent.com/14894380/146818134-d6940a50-bc8a-4bbc-98dc-3dfb230f3c4e.png)